scrapers/orgs/main.go

```diff
@@ -0,0 +1,25 @@
+package orgs
+
+import (
+	"fmt"
+)
+
+func main() {
```

Comment:
Move this logic to a custom function in `scrapers` and then call in from`main.go`. Also add the updated data to `backup.sql`.